packages/gitmoji-changelog-core/src/parser.js

```diff
@@ -26,17 +26,41 @@ function parseSubject(subject) {
   }
 }
 
-function getCommitGroup(emojiCode) {
-  const group = groupMapping.find(({ emojis }) => emojis.includes(emojiCode))
+function getMergedGroupMapping(customGroupMapping) {
+  if (!customGroupMapping) return groupMapping
+  const newCategories = customGroupMapping.filter(cg => {
+    return !groupMapping.some(g => g.group === cg.group)
+  })
+
+  const overridedCategories = groupMapping.map(group => {
+    const customGroup = customGroupMapping.find(cg => cg.group === group.group)
+    return customGroup || group
+  })
+
+  const miscellaneousIndex = overridedCategories.findIndex(g => g.group === 'misc')
+  const miscellaneousCategory = overridedCategories[miscellaneousIndex]
+
+  overridedCategories.splice(miscellaneousIndex, 1)
```

Comment:
This is a strange line, what the purpose ?

Reply:
This line remove miscellaneous category so I can place it after new categories

Reply:
Is the side effect that bothers you?